pkg/source/docker.go

```diff
@@ -15,7 +14,7 @@ import (
 type DockerSource struct {
 	imageRef    meta.OCIImageRef
 	cleanupFunc func() error
-	exportCmd   *exec.Cmd
+	// exportCmd   *exec.Cmd
```

Comment:
This is unused. Remove it? Or keep it and add a nolint tag for it?

Reply:
Remove it.